applications/conceptual-model-editor/src/app/diagram/visualization.tsx

```diff
@@ -522,18 +581,19 @@ export const Visualization = () => {
                     />
                     <Panel position="top-right">
                         <div className="flex flex-row-reverse">
-                            <div className="cursor-help" title="add class to canvas by clicking and holding `alt`">
+                            <div className="cursor-help" title="Add class to canvas by clicking and holding `alt`">
                                 ℹ
                             </div>
                             <div className="mx-1">
-                                <button title="download svg of the canvas" onClick={exportCanvasToSvg}>
+                                <button title="Download svg of the canvas" onClick={exportCanvasToSvg}>
                                     🖼
                                 </button>
                             </div>
                         </div>
                     </Panel>
                     <Background gap={12} size={1} />
                 </ReactFlow>
+                {isCanvasMenuOptionsOpen && <CanvasMenuOptions openSelectClassDialog={() => openSelectClassDialog(onSelectClassesCallback)} openCreateClassDialogHandler={openCreateClassDialogOnCanvasHandler} />}                                
```

Comment:
Do not do this .. instead be more explicit:
```
{isCanvasMenuOptionsOpen ? .... : null}
```
It is better with the isCanvasMenuOptionsClose variable. 
Or even better, render it every time and just be sure that CanvasMenuOptions returns null when not visible.